ui/raidboss/data/04-sb/trial/lakshmi.js

```diff
@@ -0,0 +1,338 @@
+      netRegexFr: NetRegexes.startsUsing({ id: '249E', source: 'Lakshmi', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '249E', source: 'ラクシュミ', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '249E', source: '吉祥天女', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '249E', source: '락슈미', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      // Intermission ability. The user WILL die if they don't use Vril.
+      id: 'Lakshmi Jagadishwari',
+      netRegex: NetRegexes.ability({ id: '2342', source: 'Lakshmi', capture: false }),
+      netRegexDe: NetRegexes.ability({ id: '2342', source: 'Lakshmi', capture: false }),
+      netRegexFr: NetRegexes.ability({ id: '2342', source: 'Lakshmi', capture: false }),
+      netRegexJa: NetRegexes.ability({ id: '2342', source: 'ラクシュミ', capture: false }),
+      netRegexCn: NetRegexes.ability({ id: '2342', source: '吉祥天女', capture: false }),
+      netRegexKo: NetRegexes.ability({ id: '2342', source: '락슈미', capture: false }),
+      alarmText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'USE VRIL OR DIE',
+        },
+      },
+    },
+    {
+      id: 'Lakshmi Divine Denial',
+      netRegex: NetRegexes.startsUsing({ id: '2485', source: 'Lakshmi', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '2485', source: 'Lakshmi', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '2485', source: 'Lakshmi', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '2485', source: 'ラクシュミ', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '2485', source: '吉祥天女', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '2485', source: '락슈미', capture: false }),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Vril + Knockback',
+          de: 'Vril + Rückstoß',
+          fr: 'Vril + Poussée',
+          ja: 'エーテル + 完全なる拒絶',
+          cn: '完全拒绝',
+          ko: '락슈미 에테르 + 넉백',
+        },
+      },
+    },
+    {
+      // Nobody with a different marker should be told to stack.
+      id: 'Lakshmi Headmarker Collect',
+      netRegex: NetRegexes.headMarker({ }),
+      run: (data, matches) => {
+        data.avoidStack = data.avoidStack || [];
+        if (matches.id !== '003E')
+          data.avoidStack.push(matches.target);
+      },
+    },
+    {
+      // Activating on any use of Hand of Beauty/Grace or Alluring Arm.
+      // Head markers don't appear until the end of the castbar,
+      // and every head marker section begins with one of these abilities,
+      // so this should be perfectly safe.
+      id: 'Lakshmi Headmarker Cleanup',
+      netRegex: NetRegexes.startsUsing({ id: ['2486', '2487', '2488'], source: 'Lakshmi', capture: false }),
```

Comment:
```suggestion
      netRegex: NetRegexes.startsUsing({ id: '248[678]', source: 'Lakshmi', capture: false }),
```
Feels more readable in my opinion, but feel free to disagree here.

Reply:
Not to muddy the waters, but I could go either way on this. The shorter version is more readable but the longer version is more searchable.

Reply:
Right, my general feeling here (weakly) is that we *have* regex helpers now, so it's a little self-defeating to then just go back to basic regex syntax. `grep/findstr` will find the array format easily enough with a regex-formatted command, but it's a little tougher to tell your search "oh, and what you're looking for *might* be string-coded regex", assuming you even know that any occurrences *are* such.

In the case of Diamond Weapon, for instance, where there are ridiculous numbers of abilities that all have to be synced on one line, the collapsed syntax makes sense, but here I weakly lean toward the expanded form.